dash/webapp/pages/toggles.js

```diff
@@ -199,7 +221,10 @@ const IndexPage = () => {
                   style={{ marginRight: '6px', marginBottom: '5px' }}
                   id={`toggle-${toggle.id}`}
                 >
-                  {toggle.title}
+                  {toggle.title}{' '}
+                  <span style={{ fontSize: '12px', color: '#666' }}>
```

Comment:
`#666` – hex code of the devil

Reply:
Maybe not

> While 666 is called the "number of the beast" in most manuscripts of Revelation 13:18, a fragment of the earliest papyrus 115 gives the number as 616.

https://en.wikipedia.org/wiki/616_(number)